src/intervals/special_intervals.jl

```diff
@@ -33,6 +33,7 @@ nai() = nai(get_interval_precision()[1])
 
 isnai(x::Interval) = isnan(x.lo) || isnan(x.hi)
 
+isfinite(x::Interval) = isfinite(x.lo) && isfinite(x.hi)
 
```

Comment:
> `isinf` and `isnan` already work by Julia magic!

Last time I checked (playing together with TaylorSeries.jl, in julia 0.4) it was not working... 

> `isnan` is now the same as `isnai`. In any case, this seems to be a pretty useless thing to do, since I don't see how `Interval`s of `NaN`s will get created ever.

Probably you are right. Yet, it is part of IEEE1768... The name `isnai`, if I recall correctly, was taken from the standard.